doc/quickstart/find_data.rst

```diff
@@ -131,6 +140,74 @@ ERA5
   of both liquid and solid phases to vapor (from underlying surface and vegetation)."
   Therefore, the ERA5 (and ERA5-Land) CMORizer switches the signs of ``evspsbl`` and ``evspsblpot`` to be compatible with the CMOR standard used e.g. by the CMIP models.
 
+.. _read_native_era5_grib:
+
+ERA5 (in GRIB format available on DKRZ's Levante)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ERA5 data in monthly, daily, and hourly resolution is `available on Levante
+<https://docs.dkrz.de/doc/dataservices/finding_and_accessing_data/era_data/index.html#era-data>`__
+in its native GRIB format.
+To read these data with ESMValCore, use the root path ``/pool/data/ERA5`` with
+DRS ``DKRZ-ERA5-GRIB`` in your :ref:`user configuration file`, for example:
```

Comment:
The documentation build appears to be broken because this link was removed in #2448

Reply:
Fixed in https://github.com/ESMValGroup/ESMValCore/pull/2178/commits/a031ea9516ce1db269396d46e380fad89d1191bd